visual-js/visual/src/cli/build.ts

```diff
@@ -104,26 +88,23 @@ export const buildStatusCommand = async (
 type BuildStatusCommandOptions = {
   buildId?: string;
   customId?: string;
-  region?: VisualApiRegion;
+  region: VisualApiRegion;
 };
 const statusCommand = () =>
   new Command()
     .name('status')
     .description('Fetches status from a Sauce Visual build')
     .addOption(buildIdOption)
     .addOption(customIdOption)
-    .addOption(regionOption)
```

Comment:
Moving this out of `build.ts` will make the region unavailable as soon as it's placed after `build` sub command (like `npx visual build create --name test_build --region eu-central-1`).

That's a bit annoying but this is the way Commander works as I observed id.

When I previously worked on this, I found no way other that duplicating that `region` param in each sub-command. It would be great if you finally got a solution for this.

Reply:
I think duplicating is fine. I'll do this